frontend/src/metabase/modes/lib/actions.js

```diff
@@ -123,11 +123,18 @@ const fieldRefWithTemporalUnitForColumn = (column, unit) =>
 export function drillFilter(query, value, column) {
   let filter;
   if (isDate(column)) {
-    filter = [
-      "=",
-      fieldRefWithTemporalUnitForColumn(column, column.unit),
-      parseTimestamp(value, column.unit).format(),
-    ];
+    if (value == null) {
```

Comment:
use double equals instead of triple equals to match the other line below
https://github.com/metabase/metabase/blob/b0baa94c100dcb950ac04fa2680a69bef929b0e3/frontend/src/metabase/modes/lib/actions.js#L142